mega-linter-runner/generators/mega-linter/templates/mega-linter.yml

```diff
@@ -62,14 +63,15 @@ jobs:
       - name: MegaLinter
 
         # You can override MegaLinter flavor used to have faster performances
-        # More info at https://megalinter.io/latest/flavors/
+        # More info at https://megalinter.io/flavors/
         uses: <%= GITHUB_ACTION_NAME %>@<%= GITHUB_ACTION_VERSION %>
 
         id: ml
 
         # All available variables are described in documentation
-        # https://megalinter.io/latest/config-file/
+        # https://megalinter.io/configuration/
```

Comment:
The original would be the correct link, https://github.com/oxsecurity/megalinter/pull/2986